etc/orthography.tsv

```diff
@@ -354,4 +354,4 @@ e_?\	eˤ	1
 O\~	ʘ̃	1
 O\X	ʘχ	1
 J:	ɲː	1
-<on>a>		1
+<on>a>	a	1
```

Comment:
I guess this line serves rather as reminder that the raw data is incorrect (this is a remainder of an ill-formatted marker). We can keep this here, but should remove as soon as the upstream data is fixed.

Reply:
Yes, and regarding the ɗ-sounds, it may be good if @Tarotis checks in which languages they occur and if this is a realistic interpretation.

Reply:
The alternative would be ejectives, so we would assume the authors confused > and <

Reply:
No, that's a pretty clear case of a ill-formatted label. `<<on>a>` would be well-formatted according to https://github.com/cldf-datasets/doreco/blob/d677c9896269339841de4cc3534481209c2d61fd/cldfbench_doreco.py#L47
So the only alternative interpretation would be to drop the `a`, because content of other labels is ignored as well.

Reply:
I meant the other case I mentioned above: `d_j_<`. This can be `d_<_j` which is what I now provide in the profile, or `d_j_>` which would be palatalized ejective.

Reply:
Thanks @LinguList for making this detailed check. The sounds occur only in Goemai (Chadic) and I think you got this right, that is a palatalized alveolar implosive stop. If I see this right, this seems to be inconsistent in X-Sampa:

X-Sampa | IPA
--- | ---
d_w_< | ɗʷ
d_<_j | ɗʲ

I, too, would have expected ` d_j_<` in there. Or was that something that we implement in the conversion, and the original form was correct?